source/event.ts

```diff
@@ -5,14 +5,24 @@ import { v4 as uuidV4 } from "uuid";
  * ftrack API Event class.
  */
 export class Event {
+  private readonly _data: {
+    topic: string;
+    data: any;
+    target: string;
+    inReplyToEvent: Event | null;
+    id: string;
+    sent: Date | null;
+    source?: any;
+  };
+
   /**
    * Construct Event instance with *topic*, *data* and additional *options*.
    *
    * *topic* should be a string representing the event.
    *
    * *data* should be an object with the event payload.
    */
-  constructor(topic, data, options = {}) {
+  constructor(topic: string, data: any, options: { [key: string]: any } = {}) {
```

Comment:
some of `any` on input type can be replaced by `object`, which is slightly more accurate.
like the `data` and `options` in here.

Reply:
fixed!